nixos/modules/services/x11/window-managers/herbstluftwm.nix

```diff
@@ -11,6 +11,16 @@ in
     services.xserver.windowManager.herbstluftwm = {
       enable = mkEnableOption "herbstluftwm";
 
+      package = mkOption {
+        type = types.package;
+        default = pkgs.herbstluftwm;
+        defaultText = "pkgs.herbstluftwm";
+        example = literalExample "pkgs.herbstluftwm";
```

Comment:
The example is the same value as the `default` which seems a bit redundant.

Reply:
Should I change the default to e.g. `pkgs.unstable.herbstluftwm` ?

Reply:
Well that isn't a real thing either. What do other modules do? Omit the `example` perhaps?

Reply:
I looked at syncthing for reference:
https://search.nixos.org/options?channel=20.09&show=services.syncthing.package&from=0&size=50&sort=relevance&query=syncthing

Reply:
In my opinion you should omit the `example` here. `syncthing` also should...

Reply:
```suggestion
```